app/assets/stylesheets/layout.scss

```diff
@@ -320,86 +338,49 @@ hr {
   }
 }
 
-.assignment-repo-list {
-  .assignment-repo-list-item:not(:first-child) {
-    padding-top: 1rem;
-    margin-top: 1rem;
-    border-top: 1px solid #e3e3e3;
-
-    @media screen and (max-width: 40em) {
-      margin-bottom: 0;
-    }
-  }
+.assignment-repo-info-container {
```

Comment:
Instead of having our own custom container can we use the new stuff that Primer 6 gives us?